glib/Gir_GObject.toml

```diff
@@ -25,6 +25,7 @@ manual = [
     "GObject.Object",
     "GObject.Value",
     "GObject.EnumValue",
+    "GObject.FlagsValue",
```

Comment:
not very important, but I guess this should always have been here also before this patch? `pub struct FlagsValue(gobject_ffi::GFlagsValue);` is already manually implemented

Reply:
yes, glib/gobject lacks a lot of types in the manually implemented list.